features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/Login/Login.js

```diff
@@ -91,13 +91,13 @@ class Login extends Component {
             this.setState({isLogin: AuthManager.getUser(), loading: false});
         }).catch((error) => {
                 this.setState({messageOpen: true});
-                if(error.response.data.code === 900964){ //Error while generating a new access token.
+                if (error.response.data.code === 900964) { //Error while generating a new access token.
                     this.setState({message: "Invalid Username or Password."});
-                }else{
+                } else {
                     this.setState({message: error.message});
                 }
                 console.log(error);
-                this.setState({loading: false});
+                this.setState({loading: false});q
```

Comment:
@RenukaFernando , Please remove this un-intended character `q` if this is present in current code

Reply:
Noted. Thanks.